how_to_spin_up_an_htap_database_in_5_minutes_with_tidb_tispark.md

```diff
@@ -0,0 +1,251 @@
+
+**Congratulations! You have just deployed a TiDB cluster on your laptop!**
+
+To check if your deployment is successful:
+
+* Go to: [http://localhost:3000](http://localhost:3000) to launch Grafana with default user/password: admin/admin.
+
+    * Go to `Home` and click on the pull down menu to see dashboards of different TiDB components: TiDB, TiKV, PD, entire cluster.
+
+    * You will see a dashboard full of panels and stats on your current TiDB cluster. Feel free to play around in Grafana, e.g. `TiDB-Cluster-TiKV`, or `TiDB-Cluster-PD`.
+
+![Grafana display of TiKV metrics](media/grafana_display_of_tikv_metrics.png)
+
+<center> Grafana display of TiKV metrics </center>
+
+* Now go to TiDB-vision at [http://localhost:8010](http://localhost:8010) (TiDB-vision is a cluster visualization tool we built in-house to see data transfer and load-balancing inside your cluster).
+
+    * You can see a ring of 3 [TiKV](http://bit.ly/tikv_repo_publication) nodes. TiKV applies the Raft consensus protocol to provide strong consistency and high availability. Light grey blocks are empty spaces, dark grey blocks are Raft followers, and dark green blocks are Raft leaders. If you see flashing green bands, that represent communications between TiKV nodes.
+
+    * It looks something like this:
+
+![TiDB-vision](media/tidb_vision.png)
+
+<center> TiDB-vision </center>
+
+# Test TiDB Compatibility with MySQL
+
+As we mentioned, TiDB is MySQL compatible, which you can essentially use TiDB as MySQL slaves with instant horizontal scalability. That’s how many innovative tech companies, like [Mobike](https://www.pingcap.com/blog/Use-Case-TiDB-in-Mobike/), use TiDB.
+
+To test out this MySQL compatibility:
+
+1. Keep the `tidb-docker-compose` running, and launch a new Terminal tab or window.
+
+2. Add MySQL to the path (if you haven’t already):
+
+	```bash
+	export PATH=${PATH}:/usr/local/mysql/bin
+
+	```
+
+3. Launch a MySQL client that connects to TiDB:
+
+	```sql
+	mysql -h 127.0.0.1 -P 4000  -u root
+	```
+
+**Result:** You will see the following message, which shows that TiDB is indeed connected to your MySQL instance:
+> Note: TiDB version number may be different.
+
+```bash
+Server version: 5.7.10-TiDB-v2.0.0-rc.4-31
+```
+![TiDB Compatibility with MySQL](media/test_tidb_compatibility_with_mysql.png)
+
+<center> The Compatibility of TiDB with MySQL </center>
+
+
+# Let’s get some data!
+
+Now we will grab some sample data that we can play around with.
+
+1. Open a new Terminal tab or window and download the `tispark-sample-data.tar.gz` file.
+
+	```bash
+	wget http://download.pingcap.org/tispark-sample-data.tar.gz
+	```
+
+2. Unzip the sample file:
+
+	```bash
+	tar zxvf tispark-sample-data.tar.gz
+	```
+
+3. Change your directory to `tispark-sample-data`:
+
+	```bash
+	cd tispark-sample-data
+	```
+
+4. Inject the sample test data from sample data folder to MySQL:
+	
+	```sql
+	mysql --local-infile=1 -u root -h 127.0.0.1 -P 4000 < dss.ddl 
```

Comment:
I think we can just remove step 3 and tell them to do `mysql --local-infile=1 -u root -h 127.0.0.1 -P 4000 < tispark-sample-data/dss.ddl`

Reply:
`mysql --local-infile=1 -u root -h 127.0.0.1 -P 4000 < tispark-sample-data/dss.ddl` meet error `ERROR 2 (HY000) at line 75: File 'customer.tbl' not found (Errcode: 2 - No such file or directory)
`
and I use 
```shell
cd tispark-sample-data
./sample_data.sh
```

Reply:
Ok! :)